packages/grafana-ui/src/components/DateTimePickers/DateTimePicker/DateTimePicker.tsx

```diff
@@ -137,19 +175,20 @@ type InputState = {
 };
 
 const DateTimeInput = React.forwardRef<HTMLInputElement, InputProps>(
-  ({ date, label, onChange, isFullscreen, onOpen }, ref) => {
+  ({ date, label, onChange, isFullscreen, onOpen, showSeconds }, ref) => {
```

Comment:
Probably here too just to be on the safe side

Reply:
done! thanks!